File: 2235-capitalize-the-title/2235-capitalize-the-title.java
class Solution {
    public String capitalizeTitle(String title) {
        String[] words=title.split(" ");
        StringBuilder result=new StringBuilder();
        for(int i=0;i<words.length;i++){
            if(words[i].length()>=3){
                words[i]=words[i].substring(0,1).toUpperCase()+words[i].substring(1).toLowerCase();
            }
            else{
                words[i]=words[i].toLowerCase();
            }
            result.append(words[i]);
            if(i!=words.length-1){
                result.append(" ");
            }
        }
        return result.toString();
    }

}

File: 2235-capitalize-the-title/2235-capitalize-the-title.py
class Solution:
    def capitalizeTitle(self, title: str) -> str:
        split_str=title.split()
        for i in range(len(split_str)):
            if len(split_str[i])>=3:
                split_str[i]=split_str[i].title()
            else:
                split_str[i]=split_str[i].lower()
        return " ".join(split_str)